containers/web-server/Dockerfile-prod

```diff
@@ -4,10 +4,10 @@
 FROM node:16 AS builder
 WORKDIR /app
 
-COPY ./containers/web-server/frontend/package.json .
+COPY frontend/package.json .
```

Comment:
Shoulud this be package-lock.json, combined with an `npm ci`? Not sure which is best for deploy?

Reply:
One for @mz8i, but builds been working fine so far.